library/src/main/java/com/factual/engine/braze/BrazeEngineIntegration.java

```diff
@@ -2,65 +2,229 @@
 
 import android.content.Context;
 
-import com.factual.engine.api.FactualCircumstance;
-import com.factual.engine.FactualEngine;
-import com.factual.engine.api.FactualCircumstanceException;
+import android.text.TextUtils;
+import android.util.Log;
+import com.appboy.Appboy;
+import com.appboy.models.outgoing.AppboyProperties;
+import com.factual.engine.api.CircumstanceResponse;
+import com.factual.engine.api.FactualPlace;
+import java.util.List;
+import java.util.UUID;
 
 public class BrazeEngineIntegration {
```

Comment:
I might have missed it, but if this is a static class you should add a private constructor to make sure that it cannot be instantiated